curvefs/conf/tools.conf

```diff
@@ -35,6 +35,7 @@ s3.bucket_name=bucket
 s3.blocksize=4194304
 s3.chunksize=67108864
 s3.useVirtualAddressing=false
+s3.objectPrefix=0
```

Comment:
maybe replace 0 with false is better?

Reply:
No, there are 3 kinds of prefixs now, maybe more later. So bool isn't ok. Uint32 or Uint8?

Reply:
ok, I will add some comments here.

Reply:
done